app/styles/modules/_custom-rows.scss

```diff
@@ -16,17 +16,24 @@
   .left {
     float: left;
     max-width: 50%;
-    padding: 0 4% 0 0;
     text-align: left;
   }
 
   .right {
     float: right;
     max-width: 50%;
-    padding: 0 0 0 4%;
     text-align: right;
   }
 
+  &.centered a {
+    display: inline-block;
```

Comment:
this CSS is different from the existing behaviour for stacked links (via the media query): https://github.com/mozilla/fxa-content-server/blob/20b9a5156e28ca65897ac59d4a504034f934adb7/app/styles/modules/_custom-rows.scss#L130-L137

Can it be reused or should we just update both to keep consistent?